### DISCUSSION TITLE
QSO - Counties issue, Darkly Issue

### DISCUSSION THREAD

**Discussion by T012m3n7oR** [magicbug/Cloudlog#1182] [Q&A] [answered]
Hello Everyone!

An issue I'm running into is that the counties drop down on the QSO -> Live QSO -> General (tab) does not appear to filter by state.  Without a state entered it appears that I get a list of all of them alphabetical.  If I select a state I do not see a difference in the county dropdown.
![image](https://user-images.githubusercontent.com/48076977/133894039-8a24086a-0008-47fd-95e7-37c330ae2263.png)
![image](https://user-images.githubusercontent.com/48076977/133894062-e41e9313-7351-4efd-9bbe-5757ab575d5b.png)
Because this list is limited to 100 entries it wouldn't be possible to select all 254 counties in TX from this dropdown either. (Controllers\Qso.php -> get_county()) (Texas has the most counties in the USA, I'm noticing this because today is TX QSO Party)

There also appears to be a couple of fields that no longer follow dark mode. (Darkly)
![image](https://user-images.githubusercontent.com/48076977/133893974-e5e7d5a0-04a9-40f2-b574-6b544deeb2ee.png)

I am up to date with the master branch.
![image](https://user-images.githubusercontent.com/48076977/133894220-4f22a15c-be2b-4a0e-821b-8c45b060f2b9.png)

Are there a work around, or is there something I'm missing with the counties drop down?

Thank you.

**Comment by AndreasK79** [comment, marked as answer]
Hi.

I recently moved the css files so that a theme only consists of 2 files instead of 4. You might need to empty cache to reload the files. I moved selectize.css (for the dropdown) to the base folder and but the theme specific things for selectize in overrides.css.

When it comes to the counties, try writing the start of the county name, and it will filter. To get it to filter on state, you need to select a state. The reason I put 100 as the limit, was that loading too many would slow down the dropdown too much. 254 might work though.

I also see in the code that you need to have at least one letter to make it filer (qso.js -> $('#input_usa_state').change(function(){) Perhaps the logic needs some fine tuning. 

Let me know if any of this helps you out.

Andreas.

**Reply by T012m3n7oR** [reply]
Ahh, that did fix the darkly issue.  However it did not fix the issue with counties.  I select state of Tx, and county I want to enter KNOX.  Knox does not appear if I start typing it.

Here is a GIF of the process I was taking, should I have waited longer?

![2021-09-18_12h35_02](https://user-images.githubusercontent.com/48076977/133895942-46a6049f-8d09-4ec0-9529-b34b334b2aec.gif)

**Reply by AndreasK79** [reply]
Thanks. Something fishy is going on here. Need to recheck the code.

**Reply by AndreasK79** [reply]
@T012m3n7oR controllers/qso.php:390 -> increase value to 300. That will fix this. I'll update the code as well.

**Reply by AndreasK79** [reply]
Code is updated. @T012m3n7oR thanks for reporting this.

**Reply by T012m3n7oR** [reply]
Thank you for the quick fix! @AndreasK79

**Comment by magicbug** [comment]
CTRL+F5 to clear caches in most browsers
